config/routes.rb

```diff
@@ -1,16 +1,17 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users,
+             controllers: {
+                 sessions: 'users/sessions',
+                 registrations: 'users/registrations'
+             }
```

Comment:
✅ Has API endpoint for authentication, both registration and login 👍🏻 

* Login endpoint - **POST :** `/login`
* Registration endpoint - **POST :** `/register` or `/signup`